dataswarm/manager/dataswarm_blob_rep.h

```diff
@@ -0,0 +1,55 @@
+#ifndef DATASWARM_BLOB_REP_H
+#define DATASWARM_BLOB_REP_H
+
+#include "dataswarm_rpc.h"  /* needed for dataswarm_result_t */
+
+typedef enum {
```

Comment:
I like this approach much better, the code seems much more straightforward to me.
Although shouldn't these be `DS_BLOB_STATE_X rather than `ACTION_X`?

Reply:
Yes, I wasn't too happy with the names. If we go for state, I would prefer:

DS_BLOB_STATE_CREATED, COMMITTED, etc.

Reply:
Sure